cve_bin_tool/checkers/jetty.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""
+CVE checker for Jetty
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-10410/product_id-34824/Eclipse-Jetty.html
+
+"""
+from ..util import regex_find
+
+
+def guess_contains_jetty(lines):
+    """Tries to determine if a file includes jetty
+    """
+    footmarks = [
+        "Jetty Server",
+        "Jetty",
+        "Eclipse Jetty",
+        "Jetty version",
+    ]
+    for line in lines:
+        for footmark in footmarks:
+            if footmark in line:
+                return True
+    return False
+
+
+def get_version(lines, filename):
+    """returns version information for Jetty as found in a given file.
+    The version info is returned as a tuple:
+        [modulename, is_or_contains, version]
+
+    modulename will be jetty if Jetty is found (and blank otherwise)
+    is_or_contains indicates if the file is a copy of jetty or contains one
+    version gives the actual version number
+
+    VPkg: eclipse, jetty   
+    VPkg: jetty, eclipse
```

Comment:
eclipse, jetty is a valid vendor product pair, but jetty, eclipse returns no results in cvedb.  Should remove that second one.